### ISSUE TITLE
Hey! Will you be at this week's meetup?

### ISSUE THREAD

**Issue by mandynicole** [pdx-lettering-collective/www#4]
We had to bail at the last minute last time because Matt lost his grandmother, but I had been meaning to carve some time out to go over ideas for the site. 

For now I think just having a central place for social media/content would be good, but I'd eventually like to build it with the CMS I'm working on if you're cool with that. There's still a few more months of development left until we have something ready for official release, but perhaps it can be a working prototype along the way. The idea is that it's entirely inline-editable and you can just drop-in plugins like forums and shops (maybe if people wanted to sell originals or prints of what they did at the meetup with part of the proceeds going back into the group for supplies, etc?).

Anyway, I've set up some other Issues so we can jot things down in the meantime or if either of us end up being unable to attend. I just feel bad having this sit here empty when we could at least have a feed generated from the event hashtag / official social media accounts.

**Comment by masgrimes** [comment]
Hey Mandy! So sorry to hear, we've missed you at lettering club. 

I'm not sure about a feature list for the site, as at this moment we're very event focused. We recently switched over to Meetup, so we're going to need to use either a plugin or interface directly with the API ourselves. There are plenty of WP plugins for that which already exist, which would be way less work.

What are you thinking the value of using a custom CMS is vs something turnkey like WP? Just want to use your CMS on a project?

Are you coming on Thursday?

**Comment by mandynicole** [comment]
> What are you thinking the value of using a custom CMS is vs something turnkey like WP? Just want to use your CMS on a project?

Yeah, pretty much. Ultimately I have other projects I can use it with, though, so I don't mind either way. 

> Are you coming on Thursday?

All signs are pointing to yes at the moment.
